src/frequenz/channels/_exceptions.py

```diff
@@ -70,9 +70,9 @@
 
 
 class Error(RuntimeError):
-    """Base error.
+    """An error originated in this library.
```

Comment:
I think it has to be one of the two, but I like the first one.
```suggestion
    """An error that originated in this library.
```
```suggestion
    """An error originating in this library.
```